internal/e2e/e2e.go

```diff
@@ -62,6 +69,16 @@ func RunAll(ctx context.Context, t Testing, plugin framework.Plugin, pod *v1.Pod
 	if postbindP, ok := plugin.(framework.PostBindPlugin); ok {
 		postbindP.PostBind(ctx, nil, pod, "")
 	}
+
+	if preFilterE, ok := plugin.(framework.PreFilterExtensions); ok {
+		s = preFilterE.AddPod(ctx, nil, pod, pi, ni)
+		RequireSuccess(t, s)
+	}
+
+	if preFilterE, ok := plugin.(framework.PreFilterExtensions); ok {
+		s = preFilterE.RemovePod(ctx, nil, pod, pi, ni)
+		RequireSuccess(t, s)
+	}
```

Comment:
You don't need to check same thing twice.

```suggestion
	if preFilterE, ok := plugin.(framework.PreFilterExtensions); ok {
		s = preFilterE.AddPod(ctx, nil, pod, pi, ni)
		RequireSuccess(t, s)
		s = preFilterE.RemovePod(ctx, nil, pod, pi, ni)
		RequireSuccess(t, s)		
	}
```

Reply:
Absolutely yes...thank you for your feedback